python/tvm/relay/op/contrib/ethosu.py

```diff
@@ -852,6 +854,60 @@ def strided_slice_pattern():
     return pattern
 
 
+class AbsParams:
+    """
+    A class to extract and store parameters of Abs in an NPU friendly way
+    """
```

Comment:
The docstring should follow the same convention adopted with the other ```*Parms``` classes.

Reply:
Done